src/PackageManager/Package/NodePackage.php

```diff
@@ -28,4 +28,26 @@ public function getDescription(): ?string
     {
         return $this->description;
     }
+
+    /**
+     * @return string
+     */
+    public function getExternalLink(): string
+    {
+        return sprintf('https://www.npmjs.com/package/%s', $this->getName());
```

Comment:
But I now realize that the ``composer show`` command doesn't get you such information. Not even the long version (Besides ``source`` maybe), and that one doesn't support ``format=json``..
```bash
# composer show symfony/console

name     : symfony/console
descrip. : Symfony Console Component
keywords :
versions : * v4.1.4
type     : library
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
source   : [git] https://github.com/symfony/console.git ca80b8ced97cf07390078b29773dc384c39eee1f
dist     : [zip] https://api.github.com/repos/symfony/console/zipball/ca80b8ced97cf07390078b29773dc384c39eee1f ca80b8ced97cf07390078b29773dc384c39eee1f
names    : symfony/console

autoload
psr-4
Symfony\Component\Console\ => .
exclude-from-classmap

requires
php ^7.1.3
symfony/polyfill-mbstring ~1.0

requires (dev)
psr/log ~1.0
symfony/config ~3.4|~4.0
symfony/dependency-injection ~3.4|~4.0
symfony/event-dispatcher ~3.4|~4.0
symfony/lock ~3.4|~4.0
symfony/process ~3.4|~4.0

suggests
psr/log-implementation For using the console logger
symfony/event-dispatcher
symfony/lock
symfony/process

conflicts
symfony/dependency-injection <3.4
symfony/process <3.3
```
That's too bad, especially since a lot of composer.json's include a ``homepage`` field, but as it's not mandatory composer obviously can't rely on it..

Reply:
Interesting - never realized that!
The following command seems to fetch the compsoer.json homepage:
```bash
composer browse --homepage --show symfony/console
```

Maybe we should keep the current "basic" feature and extend it in a future PR, if needed?
Seems like a bit more magic happening here. And of cours inconsistency between the package managers..

Reply:
Good idea, I created an issue to check this (#13).
( By the way, shouldn't you link #2 somewhere in here since this PR would solve that issue? ;-) )